qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/ls/QuteSupportForTemplateDelegateCommandHandler.java

```diff
@@ -311,4 +318,27 @@ private static QuteJavaDefinitionParams createQuteJavaDefinitionParams(List<Obje
 		return params;
 	}
 
+	private static WorkspaceEdit generateMissingJavaMember(List<Object> arguments, String commandId,
+			IProgressMonitor monitor)
+			throws JavaModelException, CoreException {
+
+		// Create java definition parameter
+		GenerateMissingJavaMemberParams params = createGenerateMissingJavaMemberParams(arguments, commandId);
+		// Return file information from the parameter
+		return QuteSupportForTemplate.getInstance().generateMissingJavaMember(params, JDTUtilsLSImpl.getInstance(),
+				monitor);
+	}
+
+	private static GenerateMissingJavaMemberParams createGenerateMissingJavaMemberParams(List<Object> arguments,
+			String commandId) {
+
+		Map<String, Object> obj = getFirst(arguments);
+		if (obj == null) {
+			throw new UnsupportedOperationException(String
+					.format("Command '%s' must be called with one QuteTemplateCodeActionParams argument!", commandId));
+		}
+		return GsonInstance.getInstance().fromJson(GsonInstance.getInstance().toJson(obj),

```

Comment:
Have you the intention do do tha for other create method because it is not consistent with other create method.

Reply:
I am planning on converting it over to the same strategy as the other methods. I did this because I got lazy when I was coding and wanted to focus on the JDT code and the interface.